### DISCUSSION TITLE
Change character's name

### DISCUSSION THREAD

**Discussion by vitmaubra** [diasurgical/DevilutionX#5845] [Q&A]
Hi,

in the old days we could change a multiplayer's character name through Battle.net. Is the any to do this within DevilutionX?

Best,
Vitor

**Comment by AJenbo** [comment]
no

**Comment by vitmaubra** [comment]
Hi AJ,

thank you.

For those who have the GOG version of the game, changing the character's name is still possible. Just choose "Battle.net" on the multiplayer menu, select the char whose name you want to change, and create a new account with the name you want your characters to be changed into.

Best,
Vitor
